widgetssdk/src/main/java/com/glia/widgets/view/head/controller/ServiceChatHeadController.kt

```diff
@@ -128,6 +129,7 @@ internal class ServiceChatHeadController(
         }
     }
 
+
```

Comment:
NIT. I think this extra line is not needed.

Reply:
done